lib/utils/host/hostusers.go

```diff
@@ -59,6 +66,13 @@ func UserAdd(username string, groups []string) (exitCode int, err error) {
 	if len(groups) != 0 {
 		args = append(args, "--groups", strings.Join(groups, ","))
 	}
+	if uid != "" {
+		args = append(args, "--uid", uid)
+	}
+	if gid != "" {
+		args = append(args, "--gid", gid)
```

Comment:
According to `useradd` man page, `--gid` specifies "The group name or number of the user's initial login group." According to the documentation that we give to the users, only numeric GIDs are supported. If we passed a non-numeric string here, it may have opened a behavior that is not documented. Can we validate this data somewhere in our system and/or document where we _expect_ it to represent a number, and when we _enforce_ it to represent a number?

Reply:
I've made it so an int is expected when its created by tctl, i dont know if it makes sense to do it elsewhere in the backend

Reply:
Can these values also appear in the Teleport configuration file? If not, then sanitizing it in `tctl` is probably enough; orherwise, I'd also validate it on server startup.

Reply:
No it'll only come from the users traits